workflow/scripts/sleuth-init.R

```diff
@@ -45,10 +45,6 @@ samples_out <- if(!is.null(model[["full"]])) {
 # store design matrix
 write_rds(samples_out, file = snakemake@output[["designmatrix"]])
 
-# remove all columns which have only NA values
-samples <- samples %>%
-	    select_if(function(col) !all(is.na(col)))
-
```

Comment:
Are we sure we can safely remove this without any side-effects? Would be great to have a comment here, explaining why we do this filtering... :sweat_smile:

Reply:
For my run it worked fine. The goal with this entire PR was the fact that the log count matrix couldn't be compared across different models since they would miss rows.

Reply:
Yeah, it probably was initially done to keep the reported table(s) clean. But with datavzrd this now restricts to explicitly named columns for the report, anyways. So i guess this should be alright, especially if all of the tests properly pass.